get-started/configuration.md

```diff
@@ -6,13 +6,26 @@ publish: true
 position: 30
 ---
 
-## Configuration
+## HTTPS Configuration
 
-The Fiddler Everywhere client needs an account with administrative rights to enable secure traffic capturing (HTTPS). Without installing trust root certificate via admin account, you will be able to capture only non-secure HTTP traffic.
+By default, Fiddler Everywhere client intercepts insecure traffic (**HTTP**) only and needs an account with administrative rights to capture secure traffic (**HTTPS**). The Fiddler Everywhere client act as a man-in-the-middle (against the HTTPS traffic). To achieve that, the proxy must generate a root certificate and use that root certificate to create multiple end-entity certificates, one for each intercepted HTTPS site. 
+
+Generate and trust the root certificate via the options provided in [**_Settings > HTTPS_**]({%slug decrypt-https-traffic%}) section. For specific related to enabling HTTPS capturing on the different operating system see the links below:
 
 - For __Windows OS__ follow the steps described in [HTTPS capturing for Windows]({%slug decrypt-https-traffic%}#https-capturing-for-windows) documenation section.
 
 - For __Mac OS__ follow the steps described in [HTTPS capturing for Windows]({%slug decrypt-https-traffic%}#https-capturing-for-mac-os) documenation section.
 
 - For __Linux OS__ follow the steps described in [HTTPS capturing for Windows]({%slug decrypt-https-traffic%}#https-capturing-for-linux) documenation section.
 
+## Network Settings
+
+The **Connections** submenu located in [**_Settings >  Connections_**]({%slug connections-submenu%}) exposes options to control network-related settings With the initial setup Fiddler Everywhere uses port **8866** and acts as a system proxy on startup. Additionally, the client can be configured to listen for requests from other computers or devices (for example, an Android device). 
```

Comment:
1) Missing period (.) after the first sentence (before With)
2) Comma between "setup" and "Fiddler"